KimFom.ShiftLogger/ShiftLoggerApi/ShiftLoggerApi/Models/Shift.cs

```diff
@@ -0,0 +1,13 @@
+using System.ComponentModel.DataAnnotations;
+
+namespace ShiftLoggerApi.Models;
+
+public class Shift
+{
+    [Key]
+    public int Id { get; set; }
+    public string? Name { get; set; }
```

Comment:
Looking good! Quick question about this model. Should the name ever be null?

Reply:
Lol nope. I was just trying to remove those warnings. The name shouldn't be null.